rocky/tests/conftest.py

```diff
@@ -1,29 +1,103 @@
 import json
 from pathlib import Path
-from unittest.mock import patch, MagicMock
-
+from unittest.mock import MagicMock
 import pytest
+import binascii
+from os import urandom
 from django.contrib.auth.models import Permission, Group
 from django.contrib.messages.middleware import MessageMiddleware
 from django.contrib.sessions.middleware import SessionMiddleware
 from django_otp import DEVICE_ID_SESSION_KEY
 from django_otp.middleware import OTPMiddleware
-
+from unittest.mock import patch
 from octopoes.models import DeclaredScanProfile, ScanLevel, Reference
 from octopoes.models.ooi.findings import Finding
 from octopoes.models.ooi.network import Network
 from rocky.scheduler import Task
-from tools.models import Organization, OrganizationMember, OOIInformation, Indemnification
-from tools.models import GROUP_REDTEAM, GROUP_ADMIN
+from tools.models import OrganizationMember, OOIInformation, Organization, Indemnification
+from tools.models import GROUP_REDTEAM, GROUP_ADMIN, GROUP_CLIENT
+
+
+def create_super_user(django_user_model, email, password, name, device_name):
+    user = django_user_model.objects.create_superuser(email=email, password=password)
+    user.full_name = name
+    user.is_verified = lambda: True
+    user.save()
+    device = user.staticdevice_set.create(name=device_name)
+    device.token_set.create(token=binascii.hexlify(urandom(8)).decode())
+    return user
+
+
+def create_user(django_user_model, email, password, name, device_name):
+    user = django_user_model.objects.create_user(email=email, password=password)
+    user.full_name = name
+    user.is_verified = lambda: True
+    user.save()
+    device = user.staticdevice_set.create(name=device_name)
+    device.token_set.create(token=binascii.hexlify(urandom(8)).decode())
+    return user
+
+
+def create_organization(name, organization_code):
+    katalogus_client = "katalogus.client.KATalogusClientV1"
+    octopoes_node = "octopoes.connector.octopoes.OctopoesAPIConnector.create_node"
+    with patch(katalogus_client), patch(octopoes_node):
+        return Organization.objects.create(name=name, code=organization_code)
+
+
+def create_member(user, organization):
```

Comment:
I think it would also be better to add a `status=OrganizationMember.STATUSES.ACTIVE` argument that we can use in `new_member` and `blocked_member`.

Reply:
You mean create a fixture for active users?

Reply:
Yes I already know where you are going with it, also fix the test for checking active member.